### PR TITLE
Add precision setting and a rename suggestion

### PR DESCRIPTION
**Pull request by 7omas**
Added setting to allow user to get millisecond precision.

tho it is internally called selectedEventIndex in Ontime, that would seem to suggest the event currently chosen in the editor and not the event currently running. So I've named it activeEventIndex. Maybe a different name would be better?

### CODE REVIEW

**Comment by 7omas** [comment]
I've been getting Script Error in Chataigne recently. It is currently says "Line 77, column23 : Unknown function 'set'"
It happens when starting or stopping events. When activating the input log for the module the error happens only after the first event message (eventNow). But all values are added as expected..
Any ideas what might be causing that?

**Comment by alex-Arc** [comment]
> tho it is internally called selectedEventIndex in Ontime, that would seem to suggest the event currently chosen in the editor and not the event currently running. So I've named it activeEventIndex. Maybe a different name would be better?

I think we call it the loaded event also, could that work?

File: module.json [hunk 1]
```diff
@@ -119,24 +132,25 @@
         "readOnly": true
       }
     },
-    "Next Event": {
+    "messageTimer": {
```
Comment (alex-Arc) [inline module.json]: I think something is going wrong in your merging
make sure to that the main branch is up to date and then create a branch from that to work on

Comment (alex-Arc) [inline module.json]: This is why line 77 complains 

```suggestion
    "Next Event": {
```

**Comment by 7omas** [comment]
The idea behind Active is that an event can be loaded or playing.
Using that description does not imply either.

**Comment by 7omas** [comment]
This PR is included in the branch PR that was just made.

**Comment by 7omas** [comment]
This PR is included in the branch PR that was just made.

**Pull request closed without merging**